.gitignore

```diff
@@ -2,3 +2,5 @@ node_modules
 npm-debug.log
 /lib
 /test/coverage
+*~
+.DS_Store
```

Comment:
Oh! @a-ignatov-parc Maybe you could add this to your global `.gitignore` config file? 😄

Reply:
Good idea! Never knew about global `.gitignore`. Will remove this change next week when I get to the computer 👍